comparative-population-studies.csl

```diff
@@ -137,20 +137,9 @@
       <if variable="DOI">
         <text variable="DOI" prefix="[https://doi.org/" suffix="]"/>
       </if>
-      <else-if type="article-newspaper">
-        <group delimiter=", ">
-          <text variable="URL" prefix="[" suffix="]"/>
-          <date variable="accessed" delimiter=".">
-            <date-part name="day" form="numeric"/>
-            <date-part name="month" form="numeric"/>
-            <date-part name="year"/>
-          </date>
-        </group>
-      </else-if>
```

Comment:
@bwiernik, did you remove this format for newspaper articles intentionally? The example at http://www.comparativepopulationstudies.de/index.php/CPoS/pages/view/AGL_E has " [http://nyti.ms/1mk7Z3u, 18.8.2014].", so don't you still need it (but move the square brackets up to the group)?